docs/src/references/agent/operation-workflow.md

```diff
@@ -379,6 +433,8 @@ For some action, notably a device `restart`, the handlers are limited to one:
 Currently, here are the available actions:
 
 - `restart` triggers a reboot of the device
```

Comment:
With the ability to trigger sub commands, the `restart` action can be deprecated.